src/ai/AIPlayer.h

```diff
@@ -44,6 +44,9 @@ class AIPlayer
 
     /// Zugriff auf die GameCommands, um diese abarbeiten zu können
     const std::vector<gc::GameCommandPtr>& GetGameCommands() const { return gcs; }
+    // access to ais CommandFactory
+    const AIInterface getAIInterface() const { return aii; }
```

Comment:
You want a reference here not a copy

Reply:
Done, somehow its exhausting to care of that, as daily java developer...